### PR TITLE
Remove fsio part 2

### PR DESCRIPTION
**Pull request by mck1117**
#3865
#2928

- remove misc fsio references in configs/build script
- remove FsioState
- remove tests

### CODE REVIEW

File: firmware/config/engines/me7pnp.cpp [hunk 1]
```diff
@@ -126,10 +126,6 @@ void vag_18_Turbo() {
 
 	//ETB Settings
 
-#if EFI_FSIO
-	// todo lua setFsio (14, GPIOF_13, "1");
-	// todo lua setFsioExt (3, GPIOE_0, "0.15 90 coolant 120 min max 90 - 30 / 0.8 * +", 25);
```
Comment (rusefillc) [inline firmware/config/engines/me7pnp.cpp:131]: these comments have value please keep those

**Review by rusefillc — CHANGES_REQUESTED** [review]

File: firmware/config/engines/toyota_jzs147.cpp [hunk 2]
```diff
@@ -123,11 +123,6 @@ void setToyota_2jz_vics() {
 	strcpy(engineConfiguration->engineCode, "2JZ");
 	strcpy(engineConfiguration->vehicleName, "VVT example");
 
-
-	// todo: these magic values would be hardcoded once we find out proper magic values
```
Comment (rusefillc) [inline firmware/config/engines/toyota_jzs147.cpp:127]: this comment has value

File: firmware/config/engines/me7pnp.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/config/engines/me7pnp.cpp:131]: this file does not have value

Comment (rusefillc) [reply firmware/config/engines/me7pnp.cpp:131]: let's move that single line to b6 config to note that radiator GPPWM could be lua

Comment (rusefillc) [reply firmware/config/engines/me7pnp.cpp:131]: as for file yes we can remove under separate PR

File: firmware/config/engines/custom_engine.cpp [hunk 3]
```diff
@@ -344,9 +344,6 @@ void setTle8888TestConfiguration() {
 	// IN2 PF14
 	// SF  PF11
 #if defined(STM32_HAS_GPIOF) && STM32_HAS_GPIOF
-#if EFI_FSIO
-	// todo lua setFsio(14, GPIOF_13, "1");
```
Comment (rusefillc) [inline firmware/config/engines/custom_engine.cpp:348]: i wonder what is that line about. today I am not excited about deciding if it's needed or not

Comment (mck1117) [reply firmware/config/engines/custom_engine.cpp:348]: well:
- it's in a configuration called `test`
- that line hasn't been touched in 6 months, and nothing has blown up

File: firmware/config/engines/me7pnp.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/config/engines/me7pnp.cpp:131]: https://github.com/rusefi/rusefi/pull/3878

**Pull request merged**